Sources/PathKit.swift

```diff
@@ -250,6 +256,60 @@ extension Path {
 
     return pathExtension
   }
+
+  /// Returns the relative path necessary to go from `base` to `self`.
+  ///
+  /// Both paths must be absolute or relative paths.
+  /// - throws: Throws an error when the path types do not match, or when `base` has so many parent path components
+  ///           that it refers to an unknown parent directory.
+  public func relativePath(from base: Path) throws -> Path {
```

Comment:
```suggestion
  public func relative(from base: Path) throws -> Path {
```

What do you think about naming this `relative(from:)`, I'm thinking about the consistency with `abbreviate()`, `symlinkDestination()`, `symlink(_:)`, `link(:)`, `home`, `temporary` etc for which the "Path" isn't explicitly in the name.

Reply:
I'm also slightly wondering if it might be better to do it the other way (which is how Python's pathlib has it `PurePath.relative_to(*other)`). I think it may be a bit easier to reason about and understand as a user of PathKit.

This is how you've instinctively written the tests and made a function which reversed the order.

```suggestion
  public func relative(to other: Path) throws -> Path {
```